frontend/apps/web/components/jobs/SchemaTable/SchemaTableToolBar.tsx

```diff
@@ -95,37 +111,63 @@ export function SchemaTableToolbar<TData>({
-          <Button
-            disabled={isBulkApplyDisabled}
-            type="button"
-            variant="outline"
-            className={cn(isBulkApplyDisabled ? undefined : 'border-blue-600')}
-            onClick={() => {
-              table.getSelectedRowModel().rows.forEach((r) => {
-                form.setValue(
-                  `mappings.${r.index}.transformer`,
-                  bulkTransformer,
-                  {
-                    shouldDirty: true,
-                    shouldTouch: true,
-                    shouldValidate: false, // this is really expensive, see the trigger call below
-                  }
+          <div className="flex flex-row gap-1">
+            <Button
+              disabled={isBulkApplyDisabled}
+              type="button"
+              variant="outline"
+              className={cn(
+                isBulkApplyDisabled ? undefined : 'border-blue-600'
+              )}
+              onClick={() => {
+                table.getSelectedRowModel().rows.forEach((r) => {
+                  form.setValue(
+                    `mappings.${r.index}.transformer`,
+                    bulkTransformer,
+                    {
+                      shouldDirty: true,
+                      shouldTouch: true,
+                      shouldValidate: false, // this is really expensive, see the trigger call below
+                    }
+                  );
+                });
+                setBulkTransformer(
+                  convertJobMappingTransformerToForm(
+                    new JobMappingTransformer()
+                  )
                 );
-              });
-              setBulkTransformer(
-                convertJobMappingTransformerToForm(new JobMappingTransformer())
-              );
-              form.trigger('mappings'); // trigger validation after bulk updating the selected form options
-              table.resetRowSelection(true);
-            }}
-          >
-            <CheckIcon />
-          </Button>
+                form.trigger('mappings'); // trigger validation after bulk updating the selected form options
+                table.resetRowSelection(true);
+              }}
+            >
+              <CheckIcon />
+            </Button>
+            {isBulkApplyDisabled &&
+              hasSelectedRows &&
+              !isTransformerAllowed(allowedTransformers, transformer) && (
+                <TooltipProvider>
+                  <Tooltip delayDuration={0}>
+                    <TooltipTrigger asChild>
+                      <div>
+                        <ExclamationTriangleIcon className="h-4 w-4  dark:text-yellow-400 text-yellow-600" />
+                      </div>
+                    </TooltipTrigger>
+                    <TooltipContent>
+                      <p>
+                        Unable to apply selected transformer in bulk due to
+                        current row selection resulting in no valid overlapping
+                        transformers.
```

Comment:
I think we can reword to be simpler and shorter. Something like: 

"Can't apply the Bulk Transformer. The rows you selected don't have any overlapping Transformers."

or

"Can't apply the Bulk Transformer because the rows you selected don't have any Transformers in common."